### PR TITLE
Add ad blocker warning

### PR DESCRIPTION
**Pull request by dreampulse**

### CODE REVIEW

File: src/app/container/cart-page.js [hunk 1]
```diff
@@ -84,9 +86,20 @@ const CartPage = ({
...
 }) => {
+  const hasAdblocker = useHasAdblocker()
+
   const numModels = modelsWithConfig.length
   const hasModels = numModels > 0
 
+  const notificationSection = () => (
+    <Section>
+      <Notification
+        message="You use an ad blocker. Paypal payment won't work. Please use credit cart payment or deactivate your ad blocker."
```
Comment (dreampulse) [inline src/app/container/cart-page.js]: @Dansken need to provide a text

File: src/app/container/cart-page.js [hunk 2]
```diff
@@ -56,6 +56,8 @@ import deleteIcon from '../../asset/icon/delete.svg'
 import copyIcon from '../../asset/icon/copy.svg'
 import backIcon from '../../asset/icon/back.svg'
 import zoomInIcon from '../../asset/icon/zoom-in.svg'
+import useHasAdblocker from '../hook/use-has-adblocker'
+import Notification from '../component/notification'
 
 const CartPage = ({
   modelsWithConfig,
```
Comment (ranzwertig) [inline src/app/container/cart-page.js]: the paypal script is loaded at the beginning, isn't it? So why do we show the notification on the cart page. The user would need to refresh the page and everything is lost?

File: src/app/hook/use-has-adblocker.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import {useState, useEffect} from 'react'
+
+const useHasAdblocker = () => {
+  const [hasAdblocker, setAdblocker] = useState(false)
+
+  useEffect(() => {
+    // Source: https://github.com/nsi88/adblockdetect/blob/master/index.js
+
+    const blockedElement = global.document.createElement('div')
```
Comment (ranzwertig) [inline src/app/hook/use-has-adblocker.js]: did you test this with different blockers? Is this best practice, or should we directly test if paypal works.

File: src/app/container/cart-page.js (hunk 2 above)
Comment (dreampulse) [reply src/app/container/cart-page.js]: Right now i'm working on a pr which allows refreshing of the page. Mikkel don't want to "warn" the user that early.

File: src/app/hook/use-has-adblocker.js (hunk 3 above)
Comment (dreampulse) [reply src/app/hook/use-has-adblocker.js]: I just googled for some best practices and they are all pretty similar to that approach. But do you have a suggestion for a better way?

Comment (ranzwertig) [reply src/app/hook/use-has-adblocker.js]: not really but we should test this before merging, did you test with different common ad blockers?

Comment (dreampulse) [reply src/app/hook/use-has-adblocker.js]: I did a test with ublock on chrome and with Adblock Plus on firefox

**Pull request merged**